cassovary-core/src/main/scala/com/twitter/cassovary/graph/node/SynchronizedDynamicNode.scala

```diff
@@ -22,7 +22,9 @@ import scala.collection.mutable.ArrayBuffer
 class SynchronizedDynamicNode(val id: Int) extends DynamicNode {
   protected val inEdges = new ArrayBuffer[Int]
   protected val outEdges = new ArrayBuffer[Int]
-  val DELETED_MARKER = 0
+  val DELETED_MARKER = Integer.MIN_VALUE
```

Comment:
I know this was already there, but it will be better to put this in a companion object to save space.

i.e., object SynchronizedDynamicNode {
val DELETED_MARKER = ...
}